src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -82,6 +82,14 @@ struct LanguageFinder {
   const std::string& language_;
 };
 
+policy_table::Languages::const_iterator FindLanguage(
+    const policy_table::MessageLanguages& msg_languages,
+    const std::string& lang) {
+  LanguageFinder finder(lang);
+  return std::find_if(
+      msg_languages.languages.begin(), msg_languages.languages.end(), finder);
```

Comment:
@Kozoriz Here you hope that everybody who will use this function will check that iterator won't be equal container.end(). 
It's bad idea to expect such responsibility. Better to add check inside method. And maybe return an object instead of iterator?